apps/01PigLatin.js

```diff
@@ -7,11 +7,128 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer:
+  //
+  // String.prototype.toLowerCase is a method that returns the value of a called string into lowercase. One of the tests needed to be passed requires that this app should auto lowercase the word before translation.
+  //
+  //
+  // By using .toLowerCase here we ensure that the word will be entirely lower case before any other changes are made. We also limit the number of times we have to search for each vowel.
+  //
+  //
+  // Since 'a' !== 'A' we would normally need search for the uppercase and lowercase strings.
+  //
+  // Since .toLowerCase makes every letter in a string lowercase we only need to search for each vowel one time.
```

Comment:
Very good! If we had left the word as it what would be and If statement for each case of the letter.